vertx-web/src/main/java/io/vertx/ext/web/Locale.java

```diff
@@ -23,46 +23,56 @@
  */
 
 import io.vertx.codegen.annotations.VertxGen;
-import io.vertx.ext.web.impl.LocaleImpl;
+import io.vertx.ext.web.impl.ParsableLanguageValue;
 
+/**
+ * @deprecated Use {@link LanguageHeader} instead
+ */
 @VertxGen
+@Deprecated
 public interface Locale {
```

Comment:
Possible but I wonder about that. The naming loses sense and cohesion if I keep this name.
Do note I'm deprecating Locale, not removing. I'm telling people that they should not use Locale for new code but it still works for old code.
Unless vertx changes its major version number, I believe vertx should be backwards compatible. This is one of those.